babel.config.js

```diff
@@ -0,0 +1,32 @@
+module.exports = function( api ) {
+	api.cache( true );
```

Comment:
Since there's apparently a cache being used here, can we make sure we cover the cleaning of it with the current clean:... scripts or perhaps add a new one for it?

Reply:
I added a new step to the scripts `clean:babel-cache` that was added to the `start:reset` script.